Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -140,6 +140,11 @@ def audit
       no_named_args = true
       [Formula.all, Cask::Cask.all]
     else
+      if args.named.any? { |named_arg| named_arg.end_with?(".rb") }
+        odeprecated "brew audit [path ...]",
```

Comment:
Oops, for future reference we normally don't add deprecations before major or minor releases. Can either comment this out for now or just leave it if no-one else cares.

Reply:
I'd say let's keep it and revert if we get a single complaint/issue

Reply:
Seeing failures on `homebrew-cask` CI: https://github.com/Homebrew/homebrew-cask/actions/runs/3789525755/jobs/6443372033

Reply:
That might be a good reason to comment this out for now. I wouldn't be surprised if other cask taps have copied our CI configuration to an extent, and if we're seeing errors they may be too. I'm also okay waiting to hear if anyone actually opens an issue, though.

Reply:
Yeah, sorry, I didn't realize that. I think it makes sense to comment it out for now. I assume this is was only failing for us because we set `HOMEBREW_DEVELOPER`, right?

Reply:
The CI config specifically passes the path to `audit`. I just updated it to use `token` instead.

Reply:
I opened up #14295 to comment out the deprecation until a future release.